diff --git a/src/Apps/Alias.hs b/src/Apps/Alias.hs
index 7fa16a9..28f6e1a 100644
--- a/src/Apps/Alias.hs
+++ b/src/Apps/Alias.hs
@@ -9,10 +9,10 @@ battery :: String
 battery = "~/bin/batterywarn"
 
 compositor :: String
-compositor = "picom --config ~/.config/picom/picom.conf"
+compositor = "systemctl --user import-environment XAUTHORITY DISPLAY && systemctl --user restart picom.service" --hacks coz nix :p
 
 bar :: String
-bar = "~/.config/polybar/launch.sh"
+bar = "DEFAULT_NETWORK_INTERFACE=$(ip route | grep '^default' | awk '{print $5}' | head -n1) polybar -r main &"
 
 wallpaper :: String
 wallpaper = "~/.fehbg"
@@ -21,7 +21,7 @@ cursor :: String
 cursor = "xsetroot -cursor_name left_ptr &"
 
 xresource :: String
-xresource = "xrdb -remove && xrdb $HOME/xrdb/cute/cute"
+xresource = "xrdb -remove && xrdb $HOME/.Xresources"
 
 myterm :: String
 myterm = "st"
