diff --git a/src/Apps/Alias.hs b/src/Apps/Alias.hs
index 104c111..3303f3d 100644
--- a/src/Apps/Alias.hs
+++ b/src/Apps/Alias.hs
@@ -6,16 +6,16 @@
 module Apps.Alias where
 
 battery :: String
-battery = "~/bin/batterywarn"
+battery = "$HOME/bin/batterywarn"
 
 compositor :: String
-compositor = "picom --config ~/.config/picom/picom.conf"
+compositor = "systemctl --user import-environment XAUTHORITY DISPLAY && systemctl --user restart picom.service" --hacks coz nix :p
 
 bar :: String
-bar = "~/.config/polybar/launch.sh"
+bar = "DEFAULT_NETWORK_INTERFACE=$(ip route | grep '^default' | awk '{print $5}' | head -n1) polybar -r main &"
 
 wallpaper :: String
-wallpaper = "~/.fehbg"
+wallpaper = "feh --bg-fill $HOME/Wallpapers/flower.jpg"
 
 cursor :: String
 cursor = "xsetroot -cursor_name left_ptr &"
